docs/index.md

```diff
@@ -22,13 +25,13 @@ Freqtrade is a crypto-currency algorithmic trading software develop in python (3
 
 ## Features
 
- 1. Download market data: Download historical data of the exchange and the markets your may want to trade with.
- 2. Select markets: Create your list or use an automatic one based on top traded volume (not available during backtesting). You can blacklist markets you don't want to trade.
- 3. Backtest: Test your strategy on past data (based on [ohlcv](https://en.wikipedia.org/wiki/Open-high-low-close_chart) candles).
- 4. Optimize: Find the best parameters for your strategy using machining learning. You can optimize buy, sell, take profit (ROI) and stop-loss.
- 5. Run: Run the bot on exchange with simulated money (dry-run) or with real money (live).
- 6. Run using edge (optional module): The concept is to find the best historical [trade expectancy](https://www.freqtrade.io/en/latest/edge/#expectancy) by markets based on variation of the stop-loss and then allow/reject markets to trade (the sizing of the trade is based on a risk of a percentage of your capital).
- 7. Control/Monitor/Analyse: Use Telegram or a REST API (start/stop the bot, profit/loss, daily summary, current open trades results...). Futher analysis can be done as trades are saved (SQLite database).
+- Download market data: Download historical data of the exchange and the markets your may want to trade with. The historical data can be based on [OHLCV](https://en.wikipedia.org/wiki/Open-high-low-close_chart) candles or be trade ticks (for exchanges that support this).
```

Comment:
While the trade / tick part is technically correct - are you sure this should be noted on the "welcome" page?
I'd rather keep it general here - it's enough to point out that freqtrade provides means to download data you need for backtesting ... not how exatly that's downloaded.

To me, that's a technical detail regarding data download ...  especially since you can't use tick-data directly in the strategy - and you problably won't be able to in the future either, as exchanges (e.g. [binance](https://binance-docs.github.io/apidocs/spot/en/#recent-trades-list)) don't provide the last 500 candles - only the data before that - so basing any trading decision on a tick that's 500 trades old might be ... not such a good idea.

Reply:
I agree with @xmatthias the backtest is based on candles (the tick download seems to be used to workaround that it is not possible to get old ohlc)

Reply:
I added this because the sentence describes the historical data as ohlcv, which _technically_ is not correct because these are either ohlcv or tick data.

My proposal here:
"- Download market data: Download historical data of the exchange and the markets your may want to trade with." Point.

Reply:
Yes seems fine as backtest say that he is based on ohlc.